contrib/completion/bash/docker

```diff
@@ -5485,6 +5485,23 @@ _docker_wait() {
 	_docker_container_wait
 }
 
+COMPOSE_PLUGIN_PATH=$(docker info --format '{{json .ClientInfo.Plugins}}' | sed -n 's/.*"Path":"\([^"]\+docker-compose\)".*/\1/p')
+
+_docker_compose() {
+	local completionCommand="__completeNoDesc"
+	local resultArray=($COMPOSE_PLUGIN_PATH $completionCommand compose)
+	for value in "${words[@]:2}"; do
+		if [ -z "$value" ]; then
+			resultArray+=( "''" )
+		else
+			resultArray+=( "$value" )
+		fi
+	done
+	local result=$(eval "${resultArray[*]}" 2> /dev/null)
```

Comment:
If we use something like
```suggestion
	local result=$(eval "${resultArray[*]}" 2> /dev/null | grep -v '^:[0-9]*$')
```
here in order to remove the `:<number>` line, we can get rid of the `%%:*` two lines below (which I find unintuitive and a bit dangerous because it mangles every completion word which happens to include a colon (image:tag)).

Do you know what the `:4` or similar line returned by cobra means?